server.js

```diff
@@ -1,25 +1,29 @@
 #!/usr/bin/env node
 const nconf = require('nconf');
+const git = require('git-rev');
 const logger = require('./log.js');
 const app = require('./src/app');
 const env = nconf.get('env');
 
-// Check ES (if in use)
-if (env.ES_HOST && typeof env.ES_HOST === 'string') {
-  const runES = require('./src/run-es');
-  const checkES = new runES.Elastic();
-  checkES.setup()
-    .then(() => {
-      startListen();
-    });
-} else {
-  startListen();
-  logger.debug(`No (valid) Elasticsearch host provided - results will NOT be saved!`);
-}
+git.tag(function (ver) {
```

Comment:
what happens, when does not run in a git repo, but as an `npm install`-ed module?

Reply:
when installed from NPM registry (`npm i timings`) everything is OK. When installed with `git clone` + `npm i` or when creating a docker image from the cloned folder, the version is `0.0.0-development` ...

Reply:
I get that part - but where do you fall back to the package.json's version?

Reply:
i'm not falling back to package.json anymore - that _was_ in `/src/config.js` but i removed it in favor of grabbing the version with `git-rev` in `/server.js`

Reply:
so in that case, it won't work if the module is consumed as a dependency from npm, right?

Reply:
Never thought of this package being consumed as a dependency. Lesson learned! If `timings` is consumed as a dependency, git-rev will get the release of the parent package, right? How about doing a "semver check" on package.json's version and if invalid, fallback on the git release tag?

Reply:
I am not sure how it is used by others, but in the readme we have the [NPM  Install](https://github.com/godaddy/timings#npm-install) section. If you feel like it is not a use-case, probably we should do stop supporting that path. What's your take on that?